src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -68,8 +82,39 @@ public void onHungerDamage(EntityDamageEvent e) {
         }
     }
 
+    @EventHandler
+    @SuppressWarnings("ConstantConditions")
+    public void onGlassCut(PlayerInteractEvent e) {
+        final Block block = e.getClickedBlock();
+        if (block == null) return;
+
+        final Material blockType = block.getType();
+        final Location blockLocation = block.getLocation();
+
```

Comment:
Hmm, looking good so far. I think there could also be a variable for ``e.getItem()``, it is called way more times than it should be.